src/gui/sdlmain.cpp

```diff
@@ -2335,22 +2349,25 @@ void Config_Add_SDL() {
 	Prop_int* Pint;
 	Prop_multival* Pmulti;
 	Section_prop* Psection;
-	const Property::Changeable::Value always = Property::Changeable::Always;
 
-	Pbool = sdl_sec->Add_bool("fullscreen",Property::Changeable::Always,false);
-	Pbool->Set_help("Start dosbox directly in fullscreen. (Press ALT-Enter to go back)");
+	constexpr auto always = Property::Changeable::Always;
+	constexpr auto deprecated = Property::Changeable::Deprecated;
+
```

Comment:
Nice short-hand simplification!

Reply:
Hey, you added the `always` shortcut in here originally ;) It is convenient.